assets/controller_sa.yaml

```diff
@@ -1,9 +1,4 @@
 apiVersion: v1
-imagePullSecrets:
-  - name: bluemix-default-secret
-  - name: bluemix-default-secret-regional
-  - name: bluemix-default-secret-international
-  - name: icr-io-secret
```

Comment:
do we have icr-io-secret secret in the namespace where csi driver is getting deployed? Can you verify what are the secrets available if there any default secret which is used by all pods or not?

Reply:
```
gunakk@Gunas-MBP IKS % kubectl get secrets -n openshift-cluster-csi-drivers
NAME                                                TYPE                                  DATA   AGE
builder-dockercfg-426ks                             kubernetes.io/dockercfg               1      6d17h
builder-token-lrvtb                                 kubernetes.io/service-account-token   4      6d17h
default-dockercfg-d7mk9                             kubernetes.io/dockercfg               1      6d17h
default-token-vzwg8                                 kubernetes.io/service-account-token   4      6d17h
deployer-dockercfg-qn4h9                            kubernetes.io/dockercfg               1      6d17h
deployer-token-vhw4h                                kubernetes.io/service-account-token   4      6d17h
ibm-cloud-credentials                               Opaque                                2      6d17h
ibm-vpc-block-controller-sa-dockercfg-j8qdq         kubernetes.io/dockercfg               1      6d17h
ibm-vpc-block-controller-sa-token-h5fs2             kubernetes.io/service-account-token   4      6d17h
ibm-vpc-block-csi-driver-operator-dockercfg-86jhf   kubernetes.io/dockercfg               1      6d17h
ibm-vpc-block-csi-driver-operator-token-nwnzw       kubernetes.io/service-account-token   4      6d17h
ibm-vpc-block-node-sa-dockercfg-v9cg9               kubernetes.io/dockercfg               1      6d17h
ibm-vpc-block-node-sa-token-4c579                   kubernetes.io/service-account-token   4      6d17h
storage-secret-store                                Opaque                                1      6d17h
```
These are the secrets in the respective namespace.  And also in the issue mentioned by Jeff and Chris. The imagePullSecrets that are causing issue are 
```
bluemix-default-secret, bluemix-default-secret-regional, bluemix-default-secret-international, icr-io-secret
```

So, we are safe to remove these. Also, any other secret that is created like - ibm-vpc-block-node-sa-dockercfg-v9cg9   , etc are because of service account creation.